src/deploy/AllocatorInit.sol

```diff
@@ -0,0 +1,203 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.8.0;
+
+import { DssInstance } from "dss-test/MCD.sol";
+import { AllocatorSharedInstance, AllocatorNetworkInstance } from "./AllocatorInstances.sol";
+
+interface WardsLike {
+    function wards(address) external view returns (uint256);
+    function rely(address) external;
+    function deny(address) external;
+}
+
+interface RolesLike {
+    function setIlkAdmin(bytes32, address) external;
+    function setUserRole(bytes32, address, uint8, bool) external;
+    function setRoleAction(bytes32, uint8, address, bytes4, bool) external;
+}
+
+interface RegistryLike {
+    function file(bytes32, bytes32, address) external;
+}
+
+interface VaultLike {
+    function ilk() external view returns (bytes32);
+    function nst() external view returns (address);
+    function file(bytes32, address) external;
+    function init() external;
+    function draw(uint256) external;
+    function wipe(uint256) external;
+}
+
+interface BufferLike {
+    function approve(address, address, uint256) external;
+}
+
+interface SwapperLike {
+    function swap(address, address, uint256, uint256, address, bytes calldata) external;
+}
+
+interface DepositorUniV3Like {
+    struct LiquidityParams {
+        address gem0;
+        address gem1;
+        uint24  fee;
+        int24   tickLower;
+        int24   tickUpper;
+        uint128 liquidity;
+        uint256 amt0Desired;
+        uint256 amt1Desired;
+        uint256 amt0Min;
+        uint256 amt1Min;
+    }
+
+    struct CollectParams {
+        address gem0;
+        address gem1;
+        uint24  fee;
+        int24   tickLower;
+        int24   tickUpper;
+    }
+
+    function deposit(LiquidityParams memory) external returns (uint128, uint256, uint256);
+    function withdraw(LiquidityParams memory, bool) external returns (uint256, uint256);
+    function collect(CollectParams memory) external returns (uint256, uint256);
+}
+
+interface KissLike {
+    function kiss(address) external;
+}
+
+struct AllocatorConfig {
+    uint256 debtCeiling;
+    address allocatorProxy;
+    uint8 facilitatorRole;
+    uint8 automationRole;
+    address facilitator;
+    address keeper;
```

Comment:
We probably need to check with Techops that they will indeed run the automation contracts sequentially using the same keeper account. Maybe they'll prefer one account per automation contract to be able to run them in independent processes.

Reply:
good point, using different addresses now.